client/src/components/common/names-capture.vue

```diff
@@ -673,7 +780,25 @@ export default class NamesCapture extends Vue {
 }
 </script>
 
-<style scoped lang="sass">
-::v-deep .v-messages__message
-  line-height: 14px !important
+<style lang="scss" scoped>
+@import "@/assets/scss/theme.scss";
+
+::v-deep .v-messages__message {
+  line-height: 14px !important;
+}
+
+.main-message-style {
+  font-size: 14px;
+  color: #495057;
+}
+.label-style {
+  font-size: 16px;
+  font-weight: bold;
+  color: #212529;
```

Comment:
Save as above, this is really close to `.copy-normal`. Could get away with just applying the modified colour here and using a base text style class. All minor stuff!

Reply:
Yeah, default font size should be 1rem I think.